### PR TITLE
Kalbarri - comments and suggestions

### PR DESCRIPTION
**Pull request by gilesknight**
Consider splitting variable creation and logic over multiple lines for newcomers

### CODE REVIEW

File: kalbarri-workshop.md [hunk 1]
```diff
@@ -205,12 +205,14 @@ Boolean data is used to store true or false values. This is useful for storing t
 Boolean operations are useful for controlling the flow of programs (e.g. if X is true then do Y) and for filtering datasets based on a condition (e.g. get all cities with a population greater than 500,000). Let's experiment with boolean conditions. 
 
 ```js
-var demoBool = z == 4;
-print(demoBool);
+var demoBool = z;
```
Comment (gilesnknight) [inline kalbarri-workshop.md:208]: Consider splitting variable creation and logic over multiple lines for newcomers

File: kalbarri-workshop.md [hunk 2]
```diff
@@ -342,6 +344,7 @@ print(diff)
 
 You should see the result of calling `subtraction()` printed in the *Console*.
 
+<!--Stacking of bands will be unfamiliar to a newcomer at this point in the lab - consider a different function or shifting this to where geographic data starts to be used -->
```
Comment (gilesknight) [inline kalbarri-workshop.md:347]: Not sure I'd have any context to what `addBands()` is if I had no prior remote sensing experience - maybe another function here that's more commonplace?

File: kalbarri-workshop.md [hunk 3]
```diff
@@ -205,12 +205,14 @@ Boolean data is used to store true or false values. This is useful for storing t
 Boolean operations are useful for controlling the flow of programs (e.g. if X is true then do Y) and for filtering datasets based on a condition (e.g. get all cities with a population greater than 500,000). Let's experiment with boolean conditions. 
 
 ```js
```
Comment (gilesknight) [inline kalbarri-workshop.md:207]: Consider splitting variable creation and logic over multiple lines for newcomers

File: kalbarri-workshop.md [hunk 4]
```diff
@@ -144,8 +144,8 @@ var z = x + y;
 </details>
 <br>
 
-### Boolean
-
+### Boolean 
+<!-- Relevant to later exercises? -->
```
Comment (gilesknight) [inline kalbarri-workshop.md:148]: Not critical to remote sensing exercise later in lab - could be slimmed out saved for a later workshop

File: kalbarri-workshop.md [hunk 5]
```diff
@@ -290,7 +292,7 @@ print(squareArray);
 <br>
 
 ## Variables
-
+<!-- Consider brinin this section earlier, before boolean? -->
```
Comment (gilesknight) [inline kalbarri-workshop.md:295]: Perhaps moved to even before variable are used in Data Types

File: kalbarri-workshop.md [hunk 6]
```diff
@@ -311,7 +313,7 @@ Using variables makes code easier to organise and write. For example, if you wan
 <br>
 
 ## Functions
-
+<!-- This could be cut, if they're not writing functions in this lab they won't be applying this theory -->
```
Comment (gilesknight) [inline kalbarri-workshop.md:316]: Not critical to this lab

File: kalbarri-workshop.md [hunk 7]
```diff
@@ -421,7 +424,7 @@ Map.addLayer(ps4Img, ps4VisParams, 'PS4 Image');
 
 ### ImageCollections
 
-An `ImageCollection` in Google Earth Engine is a stack of `Image`s; it provides a structure to group together and organise related `Image`s. For example, you could create an `ImageCollection` that stores all `Image`s captured by a satellite such as Landsat, Sentinel-2, or Planet. 
+An `ImageCollection` in Google Earth Engine is a stack of `Image`s; it provides a structure to group together and organise related `Image`s. For example, you could create an `ImageCollection` that stores all the Kalbarri `Image`s captured by the Planet satellite over a 6 month period.
```
Comment (gilesknight) [inline kalbarri-workshop.md:427]: Timeseries analogy would relate better to this lab than other satellite products

File: kalbarri-workshop.md [hunk 8]
```diff
@@ -588,7 +591,7 @@ Remote sensors on satellites measure electromagnetic energy reflected or emitted
     <figcaption>Simple model of passive satellite remote sensing.</figcaption>
 </figure>
 
-Most often, when monitoring the Earth's surface using remote sensing images, we want `Image` pixels to have units of surface reflectance. Surface reflectance is the ratio of energy of incoming solar radiation to reflected solar radiation as measured by a sensor at the Earth's surface. Reflectance has values between 0 (no incoming solar radiation is reflected) to 1 (all incoming solar radiation is reflected). The roughness and albedo of an object determine how much radiation is reflected. 
+Most often, when monitoring the Earth's surface using remote sensing images, we want `Image` pixels to have units of surface reflectance. Surface reflectance is the ratio of energy of incoming solar radiation to reflected solar radiation as measured by a sensor at the Earth's surface. Reflectance has values between 0 (no incoming solar radiation is reflected) to 1 (all incoming solar radiation is reflected). The roughness and albedo of an object determine how much radiation is reflected e.g. polar ice caps have a high albedo and thus have high surface reflectance.
```
Comment (gilesknight) [inline kalbarri-workshop.md:594]: If they're not physics students this stuff could be quite unfamiliar

File: kalbarri-workshop.md [hunk 9]
```diff
@@ -703,7 +706,7 @@ print(nmKalbarriSeroja);
 Map.addLayer(nmKalbarriSeroja, {}, 'Nearmap post-TC Seroja');
 ```
 
-You can clearly see the complete damage to some roofs, partial damage to others, while some roofs remain intact. We're going to try and automatically detect damaged roofs from satellite images. 
+Start exploring the imagery on your map - you can clearly see the complete damage to some roofs, partial damage to others, and those that remained intact. We're going to try and automatically detect damaged roofs from satellite images.
```
Comment (gilesknight) [inline kalbarri-workshop.md:709]: Prompt to explore data

**Review by jmad1v07 — APPROVED** [review]
all look good. Might move programming section out as a preliminary exercise.

**Pull request merged**